tests/ignite/metrics/regression/test_fractional_absolute_error.py

```diff
@@ -98,7 +90,7 @@ def get_test_cases():
         # check multiple random inputs as random exact occurencies are rare
         test_cases = get_test_cases()
         for y_pred, y, batch_size in test_cases:
-            _test(y_pred, y, batch_size)
+            _test(y_pred, y, batch_size, device=available_device)
```

Comment:
Please rewrite this test similarly as you have done in tests/ignite/metrics/regression/test_canberra_metric.py, test_integration